tests/aclorch_ut.cpp

```diff
@@ -0,0 +1,1024 @@
+#include "ut_helper.h"
+
+extern sai_object_id_t gSwitchId;
+
+extern CrmOrch* gCrmOrch;
+extern PortsOrch* gPortsOrch;
+extern RouteOrch* gRouteOrch;
+extern IntfsOrch* gIntfsOrch;
+extern NeighOrch* gNeighOrch;
+
+extern FdbOrch* gFdbOrch;
+extern MirrorOrch* gMirrorOrch;
+extern VRFOrch* gVrfOrch;
+
+extern sai_acl_api_t* sai_acl_api;
+extern sai_switch_api_t* sai_switch_api;
+extern sai_port_api_t* sai_port_api;
+extern sai_vlan_api_t* sai_vlan_api;
+extern sai_bridge_api_t* sai_bridge_api;
+extern sai_route_api_t* sai_route_api;
+
+namespace nsAclOrchTest {
+
+using namespace std;
+
+static size_t consumerAddToSync(Consumer* consumer, const deque<KeyOpFieldsValuesTuple>& entries)
```

Comment:
As I know `static` won't work here.
It's only C keyword for local functions

Reply:
Will remove `static` keyword.